explorer/client/src/actions/helpers.ts

```diff
@@ -60,16 +61,14 @@ export function request<
           const data = normalizeData(json)
           dispatch({ type: `FETCH_${type}_SUCCEEDED`, data })
         })
-        .catch((error: Error) => {
-          dispatch({ type: `FETCH_${type}_ERROR`, error })
+        .catch(e => {
+          dispatch({ type: `FETCH_${type}_ERROR`, errors: e.errors })
```

Comment:
When the server is not available then e.errors is undefined and you will see fancy screen with errors `Unhandled Rejection (TypeError): Cannot read property 'map' of undefined...`

Steps to reproduce:
- run client only
- go to /admin/signin and click sign in button

Only signin screen is affected.

Reply:
@hellobart have you re-built `@chainlink/json-api-client`? There's some new error handling logic in there https://github.com/smartcontractkit/chainlink/pull/2894/files#diff-65d14971d6be2b4a3b655fcd66755e25